app/cli/src/main/scala/org/bitcoins/cli/CliReaders.scala

```diff
@@ -62,4 +91,31 @@ object CliReaders {
           case _ => BlockStamp.fromString(str).get
         }
     }
+
+  implicit val dlcOfferReads: Read[DLCOffer] = new Read[DLCOffer] {
+    override def arity: Int = 1
+
+    // this will be a JSON string
+    override def reads: String => DLCOffer = str => {
+      DLCOffer.fromJson(ujson.read(str))
+    }
+  }
+
+  implicit val dlcAcceptReads: Read[DLCAccept] = new Read[DLCAccept] {
+    override def arity: Int = 1
+
+    // this will be a JSON string
+    override def reads: String => DLCAccept = str => {
+      DLCAccept.fromJson(ujson.read(str))
```

Comment:
Looking at this now, in reference to my above comment about using play json, is there no reason we can't just make this `reads` call `DLCAccept.fromJson(Json.parse(str))` where `Json.parse` is a play Json call?

Reply:
same applies for the other case classes too